Exercises/3-unique.js

```diff
@@ -3,6 +3,19 @@
 // Create and return a new array without duplicate elements
 // Don't modify initial array
 
-const unique = array => [];
+const unique = array => {
+  const table = {};
```

Comment:
We do not need this auxiliary structure here

Reply:
I thought that search in hash table works faster than search in array